lib/AuditoManager.dart

```diff
@@ -1,61 +1,77 @@
 import 'package:audioplayers/audioplayers.dart';
 import 'package:flutter/foundation.dart';  // For debugPrint
+import 'storage.dart';  // Assumed to handle all database interactions
 
 class MindfulnessAudioManager {
   final AudioPlayer _audioPlayer = AudioPlayer();
-  bool isPlaying = false;
   bool loop = false;
   double volume = 1.0;
 
-  // Initialize audio player settings
-  Future<void> initializeAudio() async {
-    // Use storage.dart if there's any database interaction required
-    debugPrint("Audio Manager initialized.");
+  // Constructor with initialization logic
+  MindfulnessAudioManager() {
+    _initializeAudio();
+  }
+
+  // Initialize audio preferences and setup using storage.dart
+  Future<void> _initializeAudio() async {
+    volume = await Storage.getVolumePreference() ?? 1.0;  // Load volume from preferences
```

Comment:
1. Not how you use the storage class
2. Not how you fetch preferences

If you don't understand how to use the storage class, then either read the documentation in the comments or ask me or Vin since we both know how to roughly use it.